model/registry/cpu.yaml

```diff
@@ -0,0 +1,37 @@
+groups:
+  - id: registry.cpu
+    prefix: cpu
+    type: attribute_group
+    brief: Attributes specific to a cpu instance.
+    attributes:
+      - id: mode
+        brief: "The mode of the CPU"
+        type:
+          allow_custom_values: true
+          members:
+            - id: user
```

Comment:
mind leaving a todo note to update the enum and move applicable definitions once https://github.com/open-telemetry/build-tools/issues/192 is implemented?